src/utils/fetcher.ts

```diff
@@ -73,7 +73,11 @@ const fetcher = async (
         location?.reload()
       }
 
-      if (isGuildApiCall || resource.includes(process.env.NEXT_PUBLIC_API)) {
+      if (
+        isGuildApiCall ||
+        (process.env.NEXT_PUBLIC_API &&
```

Comment:
It would be great if we could just use these env vars without checking if they're defined or not. Maybe we could write an env parser which would throw an error if any of the required environment variables is undefined?